testing/matrix/linter/rules/rules.go

```diff
@@ -177,13 +197,35 @@ func containerImageTagLatest(object storage.StoreObject, containers []v1.Contain
 				"Can't parse an image for container",
 			)
 		}
-		if imageParts[1] == "latest" {
+
+		if imageParts[0] != defaultRegistry {
 			return errors.NewLintRuleError("CONTAINER003",
 				object.Identity()+"; container = "+c.Name,
 				nil,
-				"Image tag \"latest\" used",
+				"Image name should be "+defaultRegistry,
```

Comment:
Not `should be`. You meant something like `All images must be deployed from the same default registry - defaultRegistry`.

Reply:
ok, changed, thanks !